akvo/rsr/models/project.py

```diff
@@ -192,6 +192,11 @@ class Project(TimestampsMixin, models.Model):
                                             help_text=_(u'Determines whether this project is an '
                                                         u'RSR Impact project.'))
 
+    # Private projects
+    is_private = models.BooleanField(_(u'is private project'), default=False,
```

Comment:
I would name the field **is_public**. Then the logic for it becomes similar to is_published.